Lib/webbrowser.py

```diff
@@ -514,7 +522,7 @@ def register_standard_browsers():
                 cmd = "xdg-settings get default-web-browser".split()
                 raw_result = subprocess.check_output(cmd, stderr=subprocess.DEVNULL)
                 result = raw_result.decode().strip()
-            except (FileNotFoundError, subprocess.CalledProcessError, PermissionError, NotADirectoryError) :
+            except (FileNotFoundError, subprocess.CalledProcessError, PermissionError, NotADirectoryError):
```

Comment:
Break 95 char lines while modifying it anyway.
```suggestion
            except (FileNotFoundError, subprocess.CalledProcessError, PermissionError,
                         NotADirectoryError):
```

Reply:
Good idea, I edited it like this to be sub-80:
```python
            except (FileNotFoundError, subprocess.CalledProcessError,
                    PermissionError, NotADirectoryError):
```